src/components/spaces/views.tsx

```diff
@@ -277,7 +274,7 @@ export function SpacesPage(props: ISpacesPageProperties): ReactElement {
         {props.organization.entity.name}.
       </p>
 
-      <details className="govuk-details" role="group">
```

Comment:
keep https://github.com/alphagov/govuk-frontend/blob/b57ade99ac353a944620d818c1f89e1005935181/src/govuk/components/date-input/template.njk#L80

Reply:
Reverted back in separate commit, with an exception to the linter.